target/linux/realtek/files-5.10/drivers/net/phy/rtl83xx-phy.c

```diff
@@ -3730,6 +3943,7 @@ static int rtl8214c_phy_probe(struct phy_device *phydev)
 {
 	struct device *dev = &phydev->mdio.dev;
 	int addr = phydev->mdio.addr;
+	int ret = 0;
```

Comment:
Just be aware, that your other patch 'improve rtl8218d' also relies on this, so these two patches are 'married' because of this, change.